.fides/fides.toml

```diff
@@ -7,7 +7,7 @@ db = "fides"
 test_db = "fides_test"
 
 [credentials]
-app_postgres = {connection_string="postgresql+psycopg2://postgres:fides@fides-db:5432/fides_test"}
+app_postgres = {connection_string="postgresql+psycopg2://postgres:fides@fides-db:5432/fides"}
```

Comment:
I'm not sure why I made this change on this branch, that was a mistake - this should be looking at the running application db not the test db, and it was why we've had a failing test on this branch for awhile.